applications/GeoMechanicsApplication/custom_python/add_custom_utilities_to_python.h

```diff
@@ -15,6 +15,7 @@
 #define KRATOS_ADD_UTILITIES_TO_PYTHON_H_INCLUDED
 
 // System includes
+#include <pybind11/functional.h>
```

Comment:
For those interested, this include is needed such that python (lambda) functions can be converted to C++ ones (since we pass functions to the exported RunStage function)